routes/ProductionRequest.ts

```diff
@@ -0,0 +1,121 @@
+import { Request, Response, NextFunction } from 'express';
+import Joi from '@hapi/joi';
+import logger from '../config/logger';
+import GovDeliveryService from '../services/GovDeliveryService';
+import { DevPortalError } from '../models/DevPortalError';
+import {validateApiList} from './DeveloperApplication';
+
+export const productionSchema = Joi.object().keys({
+  primaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  secondaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  organization: Joi.string().required(),
+  appName: Joi.string().required(),
+  appDescription: Joi.string().required(),
+  statusUpdateEmails: Joi.array().items(Joi.string()).required(),
+  valueProvided: Joi.string().required(),
+  businessModel: Joi.string(),
+  policyDocuments: Joi.array().items(Joi.string()).required(),
+  phoneNumber: Joi.string().required(),
+  apis: Joi.custom(validateApiList).required(),
```

Comment:
I was considering it to be passed in as a comma delimited string since it's how the existing developer access request is handling requested apis being passed. The conditional validation already existed so it was an easy 'plug and play' decision. Making the validator handle either would be easy to do, and the array makes more sense it my head as well.